apps/edxapp/templates/services/cms/_dc_base.yml.j2

```diff
@@ -93,15 +93,15 @@ spec:
               - "-c"
               - "python manage.py {{ service_variant }} celery inspect ping -d celery@{{ celery_worker.name }}.%$(hostname)"
           initialDelaySeconds: 120
-          periodSeconds: 5
+          periodSeconds: 30
         readinessProbe:
           exec:
             command:
               - "/bin/bash"
               - "-c"
               - "python manage.py {{ service_variant }} celery inspect ping -d celery@{{ celery_worker.name }}.%$(hostname)"
           initialDelaySeconds: 60
-          periodSeconds: 3
+          periodSeconds: 10
```

Comment:
this can maybe increase a little bit more as we know this check consume a lot ?

Reply:
This is a readiness probe, not liveness.

Reply:
:+1: